src/filtrering/filtrering-mine-filter/mine-filter-rad.tsx

```diff
@@ -1,45 +1,46 @@
-import {MineFilter} from '../../ducks/mine-filter';
 import {useDispatch, useSelector} from 'react-redux';
 import {AppState} from '../../reducer';
 import {finnSideNavn, mapVeilederIdentTilNonsens} from '../../middleware/metrics-middleware';
 import {logEvent} from '../../utils/frontend-logger';
 import {velgLagretFilter} from '../../ducks/filtrering';
-import {apneMineFilterModal} from '../../ducks/mine-filter-ui';
+import {apneMineFilterModal, markerMineFilter} from '../../ducks/lagret-filter-ui-state';
 import {Radio} from 'nav-frontend-skjema';
 import RedigerKnapp from '../../components/knapper/rediger-knapp';
 import React, {RefObject, useRef} from 'react';
 import {antallFilter} from '../../components/modal/mine-filter/mine-filter-utils';
 import {ListevisningType} from '../../ducks/ui/listevisning';
+import {LagretFilter} from "../../ducks/lagretFilter";
 
 interface LagretFilterRadProps {
-    filter: MineFilter;
-    filtergruppe: string;
+    lagretFilter: LagretFilter;
+    filtergruppe: ListevisningType;
     parentDiv: RefObject<HTMLDivElement>;
 }
 
-function MineFilterRad({filter, filtergruppe, parentDiv}: LagretFilterRadProps) {
+function MineFilterRad({lagretFilter, filtergruppe, parentDiv}: LagretFilterRadProps) {
     const dispatch = useDispatch();
     const checkboxRef = useRef<HTMLDivElement>(null);
 
     const valgtLagretFilter = useSelector((state: AppState) =>
         filtergruppe === ListevisningType.minOversikt
-            ? state.mineFilterMinOversikt.valgtMineFilter
-            : state.mineFilterEnhetensOversikt.valgtMineFilter
+            ? state.mineFilterMinOversikt.valgtLagretFilter
```

Comment:
valgtMineFilter?

Reply:
unfortunatelly not, since state.mineFilterMinOversikt is type of LagretFilterUIState, which is used for all oversikter